csrc/python_frontend/python_bindings.cpp

```diff
@@ -472,6 +478,20 @@ void initNvFuserPythonBindings(PyObject* module) {
     return ss.str();
   });
 
+  py::class_<DeviceMesh> device_mesh_class(nvfuser, "DeviceMesh");
+  device_mesh_class
+      .def(
+          "__repr__",
+	  [](DeviceMesh& self) {
+            return self.toString();
+          })
+      .def(
```

Comment:
We do not support multi-dimensional meshes for now, so it is optional to add `reshape` method at this point, but it will be needed in the future